code/Report.php

```diff
@@ -218,16 +219,43 @@ protected function sanitiseClassName($class)
      * @param array $params - any parameters for the sourceRecords
      * @return int
      */
-    public function getCount($params = array())
+    public function getCount($params = array(), $limit = null)
```

Comment:
I've just double checked that `$limit = null` is correct, as opposed to something like `int $limit = 0`

https://github.com/silverstripe/silverstripe-framework/blob/4/src/ORM/Queries/SQLSelect.php#L256

tl;dr - null is the correct way to set no limit and there are no params type hints for `$limit` through the ORM as it was written pre php7